### DISCUSSION TITLE
Формат HEIC

### DISCUSSION THREAD

**Discussion by Spyker086** [totumonline/totum-ru-issues-and-discussions#509] [Как это сделать?] [answered]
Добрый день! Подскажите плиз по формату изображений heic. Пользователь прикрепил фото в этом формате, но в тотуме просмотр этого типа файлов не работает. Не поддерживается формат?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Предпросмотр мы делаем через gd. Он heic не умеет. И imagic тоже не умеет. Возможно когда в сервисах будет внешний конвертер подключен - это будет можно, но будет потреблять отдельный сервак на конвертацию.

**Reply by Spyker086** [reply]
Понятно. Спасибо
